doc/admin-guide/files/records.config.en.rst

```diff
@@ -3485,6 +3485,26 @@ Client-Related Configuration
    engines. This setting assumes an absolute path.  An example config file is at
    :ts:git:`contrib/openssl/load_engine.cnf`.
 
+TLS v1.3 0-RTT Configuration
+----------------------------
+
+.. ts:cv:: CONFIG proxy.config.ssl.server.max_early_data INT 0
```

Comment:
I think it's worthwhile here to document that tlsv1.3 must be enabled for early data to be allowed..

Reply:
Yep, will make the change.